spec/plsql/schema_spec.rb

```diff
@@ -247,7 +247,7 @@ class TestModel < TestBaseModel
       END;
     SQL
     plsql.execute <<-SQL
-      CREATE OR REPLACE PROCEDURE test_dbms_output_large(p_string VARCHAR2, p_times INTEGER)
+      CREATE OR REPLACE PROCEDURE test_dbms_output_large(p_string VARCHAR, p_times INTEGER)
```

Comment:
Hmm, why do we remove the `VARCHAR2` version and retain the `VARCHAR` only here?